src/main/java/seedu/address/logic/commands/DeleteTagCommand.java

```diff
@@ -0,0 +1,89 @@
+/** @@author: Junting **/
+package seedu.address.logic.commands;
+
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.ReadOnlyPerson;
+import seedu.address.model.person.exceptions.DuplicatePersonException;
+import seedu.address.model.person.exceptions.PersonNotFoundException;
+import seedu.address.model.tag.Tag;
+
+/**
+ * deletes a tag from all persons in addressbook.
+ */
+public class DeleteTagCommand extends Command {
+
+    public static final String COMMAND_WORD = "deleteTag";
+    public static final String COMMAND_ALIAS = "dt";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes a specific tag from all persons in addressbook.\n"
+            + "Parameters: KEYWORD\n"
+            + "Example: " + COMMAND_WORD + " friends";
+
+    private static final String MESSAGE_DELETE_TAG_SUCCESS = "Tag deleted.";
+    private static final String MESSAGE_DELETE_TAG_FAILED = "Tag deletion unsuccessful.";
+    private final String keyword;
+
+    public DeleteTagCommand(String keyword) {
+        this.keyword = keyword;
+    }
+
+    @Override
+    public CommandResult execute() throws CommandException {
+        boolean isTagDeleted;
+        try {
+            isTagDeleted = deleteTag(new Tag(keyword));
+        } catch (IllegalValueException ive) {
+            throw new CommandException(" ");
```

Comment:
You should include something in the constructor! If you don't, the user won't know what went wrong.